Mathlib/Combinatorics/Derangements/Basic.lean

```diff
@@ -16,193 +16,210 @@ import Mathlib.Logic.Equiv.Option
 /-!
 # Derangements on types
 
-In this file we define `derangements α`, the set of derangements on a type `α`.
+In this file we define `Derangements α`, the set of derangements on a type `α`.
 
-We also define some equivalences involving various subtypes of `perm α` and `derangements α`:
+We also define some equivalences involving various subtypes of `Perm α` and `Derangements α`:
 * `derangements_option_equiv_sigma_at_most_one_fixed_point`: An equivalence between
-  `derangements (option α)` and the sigma-type `Σ a : α, {f : perm α // fixed_points f ⊆ a}`.
-* `derangements_recursion_equiv`: An equivalence between `derangements (option α)` and the
-  sigma-type `Σ a : α, (derangements (({a}ᶜ : set α) : Type _) ⊕ derangements α)` which is later
+  `Derangements (Option α)` and the sigma-type `Σ a : α, {f : perm α // fixed_points f ⊆ a}`.
+* `derangements_recursion_equiv`: An equivalence between `Derangements (Option α)` and the
+  sigma-type `Σ a : α, (Derangements (({a}ᶜ : Set α) : Type _) ⊕ Derangements α)` which is later
   used to inductively count the number of derangements.
 
-In order to prove the above, we also prove some results about the effect of `equiv.remove_none`
-on derangements: `remove_none.fiber_none` and `remove_none.fiber_some`.
+In order to prove the above, we also prove some results about the effect of `Equiv.removeNone`
+on derangements: `RemoveNone.fiberNone` and `RemoveNone.fiber_some`.
 -/
 
 
 open Equiv Function
 
 /-- A permutation is a derangement if it has no fixed points. -/
-def derangements (α : Type _) : Set (Perm α) :=
+def Derangements (α : Type _) : Set (Perm α) :=
```

Comment:
This should remain lowercase

Reply:
I upcased by analogy with `PNat`, etc, contra mathport. Brought to Zulip for discussion.